src/app/index.js

```diff
@@ -76,8 +76,9 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
     if (request.status === 200) {
       var config = JSON.parse(request.responseText);
       angular.forEach(config, function(value, key) {
-         jhConfig[key] = value;
+         jhConfig[key] = value.replace("%{hostname}",window.location.host);
```

Comment:
What about extracting this logic to a separate function? Just in case we want to add more placeholders.